.env.example

```diff
@@ -0,0 +1,2 @@
+PORT=''
+SECRET=''
```

Comment:
hey @okabamac , what do we need the PORT and SECRET here for ?

Reply:
It's an example file for the .env and I guess since we sort of used this two variables currently and I needed to give an example on how to use them. Thanks